client/extensions/woocommerce/app/store-stats/store-stats-period-nav/index.js

```diff
@@ -0,0 +1,90 @@
+/** @format */
+
+/**
+ * External dependencies
+ */
+
+import React, { Fragment } from 'react';
+import PropTypes from 'prop-types';
+import { localize } from 'i18n-calypso';
+import page from 'page';
+
+/**
+ * Internal dependencies
+ */
+import HeaderCake from 'components/header-cake';
+import StatsPeriodNavigation from 'my-sites/stats/stats-period-navigation';
+import Intervals from 'blocks/stats-navigation/intervals';
+import DatePicker from 'my-sites/stats/stats-date-picker';
+
+const getGoBackFn = ( path, querystring ) => {
```

Comment:
Would `page.back` be useful here? (I haven't used it myself) or maybe [window.history.back](https://github.com/Automattic/wp-calypso/blob/112c7f3c62e8c089dd2756fe40de99fb90e50b4e/client/reader/header-back/index.jsx#L6)?

Reply:
I replaced this logic with `window.history.back`. The only place this might fail is if someone links to this page from an external source or pastes the link directly. The experience will be a bit odd, but perhaps not unexpected. Otherwise, it works well and reduces some logic and code

@greenafrican, what do you think? Modification in 827775088a6d1dbefc3cc7945dec77c4144d6395, I can easily revert